frontend/device/sony-prstux/device.lua

```diff
@@ -1,6 +1,7 @@
 local Generic = require("device/generic/device") -- <= look at this file!
 local logger = require("logger")
 local TimeVal = require("ui/timeval")
+local PluginShare = require("pluginshare")
```

Comment:
Oh, could you please also sort these as per the coding style?

https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires